test/index.test.js

```diff
@@ -420,6 +420,21 @@ describe('Sensei', () => {
         throw e
       }
     })
+
+    test('Retry 429 case', async () => {
```

Comment:
it is unclear what this test case is actually testing. i think you need these two cases:
- retry several times and succeed
- retry until failure from 429s

Reply:
Yeah, since error test only tests errors, you may need to create a new one called`retryTest`: https://github.com/adobe/aio-lib-photoshop-api/blob/47b577257b24f7af12e6fbda91b40ae155f11662/test/index.test.js#L108-L141
that checks for a failure call then a success